docs/beaver-builder/troubleshooting/miscellaneous/font-awesome-icons.md

```diff
@@ -51,7 +51,7 @@ worked, you could try adding this custom rule. See [this article](/beaver-builde
 .fas { font-weight:900 !important; }
 ```
 
-The `.fas` class is used for FA5 Solid icons.
+The `.fas` class is used for FA5 Solid icons. See [this article](https://fontawesome.com/how-to-use/on-the-web/advanced/css-pseudo-elements) for more details.
```

Comment:
I would delete this because we added that note lower down.

Reply:
Removed that.